include/quadrilateral_element.h

```diff
@@ -53,6 +54,31 @@ namespace mpm {
 //!  0      4       1
 //!
 //! </pre>
+//! Face numbering for 4-node, 8-node and 9-node Quadrilateral Element \n
+//! <pre>
+//!
+//!          F2
+//!   3 0----------0 2
+//!     |          |
+//!  F3 |          | F1
+//!     |          |
+//!     |          |
+//!   0 0----------0 1
+//!          F0
+//! </pre>
+//!
+//!
+//! Namespace containing constants of face id
+
+const std::map<unsigned, Eigen::VectorXi> face_indices_quadrilateral{
```

Comment:
This isn't even declared in the class?

Reply:
I will make it to private variable then?